deps/exokit-bindings/webglcontext/src/webgl.cc

```diff
@@ -4511,11 +4520,7 @@ NAN_METHOD(WebGLRenderingContext::GetParameter) {
       // return a int32[2]
       GLint params[2];
       glGetIntegerv(name, params);
-
-      Local<Array> arr = Nan::New<Array>(2);
-      arr->Set(0,JS_INT(params[0]));
-      arr->Set(1,JS_INT(params[1]));
-      info.GetReturnValue().Set(arr);
+      info.GetReturnValue().Set(createTypedArray<Int32Array>(2, params));
```

Comment:
(gl.getParameter returns typed arrays in Chrome.)

Reply:
Good catch.